src/treemap.ts

```diff
@@ -273,8 +274,41 @@ export class TreeMap {
       let address = getAddress(node);
       this.zoom(address);
     };
+    dom.onkeydown = e => {
+      if (!(e.target instanceof HTMLElement)) return;
+
+      let elem;
+      switch (e.key) {
+        // zoom to selection
+        case 'Enter':
+          const address = getAddress(e.target);
+          this.zoom(address);
+          return;
+        // move selection focus
+        case 'ArrowUp':
+          elem = e.target.parentElement;
+          break;
+        case 'ArrowDown':
+          elem = e.target.querySelector('.webtreemap-node');
+          break;
+        case 'ArrowLeft':
```

Comment:
somehow I was able to put focus behind the active element, e.g. got it to select tsserver.js from this view:

![image](https://user-images.githubusercontent.com/639216/63546887-b2eae380-c4df-11e9-8ffb-69563e6d55cb.png)

Reply:
Seems like some goofiness with arrowing to siblings when zoomed in?